lib/src/week_day.dart

```diff
@@ -1,3 +1,5 @@
+// Consider localizing this
```

Comment:
localization can be done on user side, since this will add unwanted overhead to this package